virocon/predefined.py

```diff
@@ -63,7 +63,7 @@ def _exp3(x, a, b, c):
     exp3 = DependenceFunction(_exp3, bounds, latex="$a + b * \exp(c * x)$")
 
     dist_description_hs = {
-        "distribution": WeibullDistribution(),
+        "distribution": WeibullDistribution(f_gamma=0),
```

Comment:
Why do you want to fix gamma to zero? That would turn it into a 2-parameter Weibull distribution. 
In DNVGL-RP-C205 (2017) a 3-parameter Weibull distribution is recommended and we provide this model here.

<img width="721" alt="Screenshot 2025-03-21 at 22 11 38" src="https://github.com/user-attachments/assets/a64cf766-e9b2-4e09-9a46-fae96833bb7c" />


```suggestion
        "distribution": WeibullDistribution(),
```

Reply:
Sure. I think I added this at some point because the lowest values are otherwise not included by the contour with method of moments.
![image](https://github.com/user-attachments/assets/14f1b541-9ae0-455e-8bff-ad438dd4ea68)
But, you are of course correct, it should be 3 parameter according to the text. Also, looking at it again now, the quantiles overall are clearly better with not fixing it:
![image](https://github.com/user-attachments/assets/ca284c10-9bcc-4029-ac9d-91e82572eb6b)